src/Common/src/Interop/Unix/System.Native/Interop.Fcntl.cs

```diff
@@ -20,6 +20,12 @@ internal static class Fcntl
 
             [DllImport(Libraries.SystemNative)]
             private static extern int FcntlCanGetSetPipeSz();
+
+            [DllImport(Libraries.SystemNative, EntryPoint="FcntlGetIsNonBlocking", SetLastError=true)]
+            internal static extern int GetIsNonBlocking(int fd);
+
+            [DllImport(Libraries.SystemNative, EntryPoint="FcntlSetIsNonBlocking", SetLastError=true)]
+            internal static extern int SetIsNonBlocking(int fd, int isNonBlocking);
```

Comment:
Should we declare isNonBlocking as a bool with [MarshalAs(UnmanagedType.Bool)]?  It'd make call sites a bit clearer I think.  Same could be done for the return type of GetIsNonBlocking.  @nguerrera, any concerns with something like that?